Add to pipenv.md/helen vu (#2)
kenny-cfg/cfg-202309-software-3: cheat_sheets/pipenv.md

More detailed instructions can be found [here](https://pipenv.pypa.io/en/latest/)

> You'll have to install the `pipenv` program first. You can do this using `pip install --user pipenv`, or `pip3 install pipenv` if you have Python3 installed. Check that it has installed by typing `pipenv --version` in the command line. You might have to restart your terminal.

Reviewer: This isn't quite right: Python 3 is the only supported version of Python these days.

However, on older Macs (for instance), the system itself uses Python 2, which means that running Python 3 uses the `python3` command, and `pip3` for pip.

Some users (especially Windows, but also sometimes Mac) will experience difficulty getting `pipenv` to run. I haven't quite got to the bottom of why, but if you try this and it fails **you should pipe up immediately**. You should get this sorted out sooner rather than later: it will be quite a big blocker for later parts of the course.

For Mac users who are having difficult getting pipenv, you may need to set the path of pipenv on your machine, see this [video](https://www.youtube.com/watch?v=Bzn_MZ0tNXU) for the following:

> * `python3 -m site --user-base` to find your user base directory of Python. Change `python3` to `python` if you have that version installed instead

Reviewer: ```suggestion
* `python3 -m site --user-base` to find your user base directory of Python. Change `python3` to `python` if you need to
```

Author: I saw I made a typo “difficult” should be “difficulty”
